tests/test_helper/many.bash

```diff
@@ -3,6 +3,22 @@ function pem() {
     echo "$PEM_ROOT/id-$1.pem"
 }
 
+# Print the X-coord of an Ed25519 public key
+function ed25519_x_coord() {
+    openssl pkey -in "$(pem "$1")" -text_pub -noout | grep "    " | awk '{printf("%s ",$0)} END { printf "\n" }' | sed '$s/\s\+//g' | sed '$s/://1g'
```

Comment:
Need to avoid using `sed` like that, unfortunately. The BSD implementation (used on Mac) has different flags than the GNU one.

```
   sed: 1: "$s/://1g": more than one number or 'g' in substitute flags
```

Can you use `tr` instead?